assets/sass/components/_issue-sidebar.scss

```diff
@@ -189,15 +189,6 @@
   display: block;
   margin-right: var(--i-bar-check-margin);
   position: relative;
-  transition: transform 0.15s ease-out;
-
-  .i-bar-item:hover & {
```

Comment:
I am really against this scale animation on hover -- if it's non-negotiable, please let me know and I will add it back